sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/objects/objects.scala

```diff
@@ -109,7 +171,7 @@ case class Invoke(
     functionName: String,
     dataType: DataType,
     arguments: Seq[Expression] = Nil,
-    propagateNull: Boolean = true) extends Expression with NonSQLExpression {
+    propagateNull: Boolean = true) extends Expression with InvokeLike with NonSQLExpression {
```

Comment:
The comment of `Invoke` doesn't have explanation for the param `propagateNull`. Can you also add it too?